.drone.yml

```diff
@@ -1,193 +1,342 @@
 kind: pipeline
-name: deb_packaging
+name: Debian 8
+type: exec
 
 platform:
   os: linux
   arch: amd64
 
+node:
+  distro: debian8
+
+# I would like to put a single "environmet" section here and do not copy it
```

Comment:
IIRC, it's possible to use an advanced feature of YAML called anchors to reference some YAML code multiple times, and that's how the Drone team suggests to reuse environment variables between pipelines.

Reply:
Used an anchor and removed comment.
Unfortunately, there is no way to reference anchors between pipelines. Hence optimization is minimal.